shared/oae/css/oae.components.css

```diff
@@ -972,9 +972,9 @@ div.oae-thumbnail i.fa {
  *      </a>
  *      <div class="oae-tile-metadata">
  *          <i class="fa fa-oae-public pull-left"></i>
- *          <h3 class="oae-threedots">
+ *          <div class="oae-tile-title oae-threedots">
```

Comment:
Rather than requiring another class in this "component", maybe we can just wrap the title in a `span` and create the CSS rules against the `span` directly